test/core/transport/binder/end2end/fuzzers/server_fuzzer.cc

```diff
@@ -0,0 +1,128 @@
+// Copyright 2021 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <grpc/grpc.h>
+
+#include "src/core/ext/transport/binder/transport/binder_transport.h"
+#include "src/core/lib/iomgr/executor.h"
+#include "src/core/lib/slice/slice_internal.h"
+#include "src/core/lib/surface/server.h"
+#include "test/core/transport/binder/end2end/fuzzers/fuzzer_utils.h"
+
+bool squelch = true;
+bool leak_check = true;
+
+static void* tag(intptr_t t) { return reinterpret_cast<void*>(t); }
```

Comment:
Use anonymous namespace?

Reply:
Same as above. I want to keep everything copied clean so that the difference stands out.